src/entities/datalist/libs/constants.ts

```diff
@@ -1,12 +1,27 @@
 import { AutoFormValueType } from "@/components/auto-field";
 
 export enum FilterFields {
```

Comment:
ℹ️ After implementing the new filtering approach, `enum FilterFields` and `const FILTERS` will become obsolete. Please ensure to remove them.